doc/subsystems/networking/usbnet_setup.rst

```diff
@@ -0,0 +1,79 @@
+.. _usb_device_networking_setup:
+
+USB Device Networking
+#####################
+
+This page describes how to set up networking between a Linux host
```

Comment:
Why limiting this to Linux, does the current CDC-ECM driver works with other OS (Windows, OS-X, ...)